dts/bindings/rtc/epson,rx8130ce.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2024 Måns Ansgariusson <mansgariusson@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: RX8130CE RTC
+
+compatible: "epson,rx8130ce-rtc"
+include: i2c-device.yaml
+properties:
+  irq-gpios:
+    type: phandle-array
+
+  clockout-frequency:
+    type: int
+    default: 0
+    description: |
+      Specify the ociillator frequency, the default is 0 (off)
+      - 0 # off
+      - 1 # 1 Hz oscillator
+      - 2 # 1 kHz oscillator
+      - 3 # 32 kHz oscillator
+    enum: [0, 1, 2, 3]
```

Comment:
Providing the frequency here in Hz is easier to read than an enum
```
enum:
  - 1
  - 1000
  - 32768
```
to disable the output, just don't provide the property :) you can use DT_PROP_OR() from the driver :)

Reply:
Great tip, will fix!